README.org

```diff
@@ -239,13 +239,18 @@ When ready, /M-x org2jekyll-publish/ to publish it.
 
 /M-x org2jekyll-publish-posts/
 
-Depending on your org-publish configuration and org2jekll, this will compulse the list of org-mode posts (*#+LAYOUT* with 'post' value) and publish them.
+Depending on your org-publish configuration and org2jekll, this will compile the list of org-mode posts (*#+LAYOUT* with 'post' value) and publish them.
```

Comment:
This is clearer indeed!

Can you please fix my original `orgjekll` typo to `org2jekyll`?
Thanks.